code/game/objects/items/crayons.dm

```diff
@@ -502,62 +502,71 @@
 			return G
 
 /obj/item/toy/crayon/red
```

Comment:
Would it be feasible to set the name on Initialize using the `crayon_color` variable?
You'd have to add case checking for things like the mime color, but I imagine it would be cleaner

Reply:
It does that already but the crafting menu only cares about the initial name.

Reply:
oh i see lol